packages/components/src/tab-panel/style.scss

```diff
@@ -32,6 +37,15 @@
 			left: 0;
 			border-bottom: 3px solid transparent;
 		}
+
+		&:focus:not(:disabled) {
+			border-bottom: none;
+			box-shadow: inset 0 -3px $theme-color;
+		}
 	}
 
 }
+
+.components-tab-panel__tab-content:focus {
```

Comment:
Why do we need this?

Reply:
It's related to the comment above from @enriquesanchez basically, if you tab in the inserter, you'll have two tab stops without focus styles if we remove the changes on this component. I think the focus styles I added are not great and I'd love some designer CSS help to improve them.

Reply:
Can we remove the tab stop that here entirely?

I'll see if I can help with the active tab focus.

Reply:
I don't think we can, at least the component (TabPanel) had very extensive a11y testing when it was built. I'd defer to others about alternatives though.

Reply:
Understood. Only asking because it used to be able to collapse but doesn't anymore, and the document sidebar has tabs but doesn't have a stop for "tab contents".

Reply:
We're not using the TabPanel component there and just a list of buttons. I can update if needed but I do believe the TabPanel had more a11y thinking in it. @diegohaz Do you have thoughts on this? (the best way to achieve tabs, can we remove the "tab content" tab stop?)

Reply:
It would be good to have some real user testing for this particular widget. But the WAI-ARIA recommendations say this:

> Tab: When focus moves into the tab list, places focus on the active tab element. When the tab list contains the focus, moves focus to the next element in the page tab sequence outside the tablist, **which is typically either the first focusable element inside the tab panel or the tab panel itself**.
>
> — https://www.w3.org/TR/wai-aria-practices/#keyboard-interaction-19

So, in theory, as long as you have a focusable element inside the tab panel, you don't need to make the container focusable.

`role="tabpanel"` elements usually have an `aria-labelledby` attribute pointing to their corresponding `role="tab"` element. Whenever you enter a tab panel with NVDA, it will announce it and its label, regardless of whether you're focusing the tab panel itself or an item inside it.

If the element has `role="tabpanel"`, VoiceOver will always move to the tabpanel when using <kbd>Control</kbd> + <kbd>Option</kbd> + Arrow keys. In this case, making it focusable or not doesn't really matter.

Reply:
Great thanks for the details here @diegohaz I believe we can try without tab stop then and test 👍